debian/buster/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM debian:buster
+MAINTAINER Alexey Kopytov <akopytov@gmail.com>
+
+# Fix missing locales
+ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
+
+# Skip interactive post-install scripts
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Don't install recommends
+RUN echo 'apt::install-recommends "false";' > /etc/apt/apt.conf.d/00recommends
+
+# Enable extra repositories
```

Comment:
Now the comment does not match to what the commend doing.

Reply:
Agree. Fixed.